### ISSUE TITLE
"Log in with Open Food Facts" and User Stats

### ISSUE THREAD

**Issue by jolesh** [openfoodfacts/openfoodfacts-hungergames#175]
### What

- I have a long term idea/suggestion:
- I think it would be nice if 
  - you could log in with your OFF account
  - you could get statistics on how much you contributed on what. And maybe get some achievements or milestones.

### Part of
- #234

**Comment by teolemon** [comment]
We could have a simple version with a Hunger Games cookie.

**Comment by alexfauquette** [comment]
@jolesh There is a counter giving you the number of annotated elements. Do you think more stuff should be added?

On my side, I think annotations reward would make more sense to be on OFF main website, because you want to reward contributions and not only contributions to hungergames

![image](https://user-images.githubusercontent.com/45398769/132914433-80db89a7-63c2-4faa-974e-49fda282f7a4.png)

**Comment by jolesh** [comment]
@alexfauquette That's right, you definitely have a point there. But then you should be able to log in so the changes are being made with you own OFF-account.

**Comment by alexfauquette** [comment]
Ok. For now, it works if you are logged in the off main site (shared cookies)

Autentication from robotoff is on the way with issue #212 and PR #290

**Comment by teolemon** [comment]
We can fetch some stats from the web using:
https://world.openfoodfacts.org/editor/teolemon.json?fields=count
https://world.openfoodfacts.org/contributor/teolemon.json?fields=count
https://world.openfoodfacts.org/photographer/teolemon.json?fields=count
